### PR TITLE
Create download_data.py

### LINKED ISSUE #4: Write the download data script

### PR DESCRIPTION
**Pull request by SonQBChau**

### CODE REVIEW

**Comment by NikitaShymberg** [comment]
Looks good! I tested the script on my end and it worked well. I just have some minor improvements that need to be made.

File: src/download_data.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# author: Son Chau
+# date: 2021-11-19
+
+"""This script prints out docopt args.
```
Comment (NikitaShymberg) [inline src/download_data.py]: This script downloads the zipped data from the given URL and saves it locally.

File: src/download_data.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+Example:
+python download_data.py --url=http://www3.dsi.uminho.pt/pcortez/wine/winequality.zip --path=./data
+"""
+
+from docopt import docopt
+from urllib.request import urlopen
+from io import BytesIO
+from zipfile import ZipFile
+
+# define main function
```
Comment (NikitaShymberg) [inline src/download_data.py]: This comment is not necessary - it doesn't tell us anything useful

File: src/download_data.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+    # download data
+    with urlopen(url) as response:
+        zf = ZipFile(BytesIO(response.read()))
+        zf.extractall(path)
+
+    print('Data downloaded to {}'.format(path))
+ 
+
+# call main function
```
Comment (NikitaShymberg) [inline src/download_data.py]: This comment is also not necessary - it doesn't tell us anything useful

File: src/download_data.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+from io import BytesIO
+from zipfile import ZipFile
+
+# define main function
+def main():
+
+    # parse arguments
+    args = docopt(__doc__)
+
+    # assign args to variables
```
Comment (NikitaShymberg) [inline src/download_data.py:25]: I would also remove this and keep these two lines under the "parse arguments" comment.

**Review by NikitaShymberg — CHANGES_REQUESTED** [review]
Just a couple of small documentation / comment changes.

**Comment by SonQBChau** [comment]
I pushed the update with fixed

**Review by NikitaShymberg — APPROVED** [review]
Looks good to me!

**Pull request merged**